src/pages/UserMyVisits.jsx

```diff
@@ -1,11 +1,158 @@
-import React from 'react';
-import { Layout } from '../components';
-import { Typography } from '@mui/material';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import Grid from '@mui/material/Grid';
+import EventAvailableIcon from '@mui/icons-material/EventAvailable';
+import isSameDay from 'date-fns/isSameDay';
+
+import { Layout, DatePicker, CustomButton } from '../components';
+import { paths } from '../config/paths';
+import { db, auth } from '../config/firebase';
+import { doc, deleteDoc } from 'firebase/firestore';
 
 export const UserMyVisits = () => {
+  const [loading, setLoading] = useState(true);
+  const [visits, setVisits] = useState([]);
+  const userId = auth.currentUser.auth.lastNotifiedUid;
```

Comment:
useState for userId, and value should be set in useEffect

Reply:
fixed